diff --git a/src/version.h b/src/version.h
index cda6b76..8baa64e 100644
--- a/src/version.h
+++ b/src/version.h
@@ -33,7 +33,7 @@ static const int DATABASE_VERSION = 70507;
 static const int PROTOCOL_VERSION = 91000;
 
 // earlier versions not supported as of Feb 2012, and are disconnected
-static const int MIN_PROTO_VERSION = 209;
+static const int MIN_PROTO_VERSION = 91000;
 
 // nTime field added to CAddress, starting with this version;
 // if possible, avoid requesting addresses nodes older than this
